bin/migration-scripts/lts-to-lts/0.16-to-0.20/playbooks/files/prisma1_to_prisma2.sql

```diff
@@ -12,8 +12,17 @@ ALTER TABLE "houston$default"."DockerImage" alter column labels type jsonb using
 ALTER TABLE "houston$default"."DockerImage" alter column env type jsonb using env::JSON;
 ALTER TABLE "houston$default"."DockerImage" RENAME COLUMN "deployment" TO "deploymentId";
 ALTER TABLE "houston$default"."User" RENAME COLUMN "localCredential" TO "localCredentialId";
-CREATE TYPE "houston$default"."Role" AS ENUM ('SYSTEM_ADMIN', 'SYSTEM_EDITOR', 'SYSTEM_VIEWER', 'WORKSPACE_ADMIN', 'WORKSPACE_EDITOR', 'WORKSPACE_VIEWER', 'DEPLOYMENT_ADMIN', 'DEPLOYMENT_EDITOR', 'DEPLOYMENT_VIEWER', 'USER');
-CREATE TYPE "houston$default"."InviteSource" AS ENUM ('SYSTEM', 'WORKSPACE');
+
+DO $$ BEGIN
+IF NOT EXISTS (SELECT 1 FROM pg_type WHERE typname = 'Role') THEN
+    CREATE TYPE "houston$default"."Role" AS ENUM ('SYSTEM_ADMIN', 'SYSTEM_EDITOR', 'SYSTEM_VIEWER', 'WORKSPACE_ADMIN', 'WORKSPACE_EDITOR', 'WORKSPACE_VIEWER', 'DEPLOYMENT_ADMIN', 'DEPLOYMENT_EDITOR', 'DEPLOYMENT_VIEWER', 'USER');
+END IF;
+
+IF NOT EXISTS (SELECT 1 FROM pg_type WHERE typname = 'InviteSource') THEN
+  CREATE TYPE "houston$default"."InviteSource" AS ENUM ('SYSTEM', 'WORKSPACE');
+END IF;
+END $$;
```

Comment:
this is the part that fixes the second-time-apply issue

Reply:
good catch!

Reply:
@sjmiller609 ty!